src/renderers/dom/client/eventPlugins/ChangeEventPlugin.js

```diff
@@ -176,7 +186,11 @@ function startWatchingForValueChange(target, targetID) {
   );
 
   Object.defineProperty(activeElement, 'value', newValueProp);
-  activeElement.attachEvent('onpropertychange', handlePropertyChange);
+  if (activeElement.attachEvent) {
```

Comment:
FYI: I'm working on a related bug now, and it appears that indeed "onpropertychange" works with attachEvent(), but doesn't with addEventListener(). At least in IE11 in IE10 Compatibility Mode. FWIW :)